packages/zowe-explorer-api/src/profiles/ZoweExplorerApi.ts

```diff
@@ -440,7 +440,11 @@ export namespace ZoweExplorerApi {
          * @returns {zowe.IIssueResponse>}
          * @memberof ICommand
          */
-        issueTsoCommand?(command: string, acctNum?: string): Promise<zowe.IIssueResponse>;
+        issueTsoCommand?(
+            command: string,
+            acctNum?: string,
+            startParams?: zowe.IStartTsoParms
+        ): Promise<zowe.IIssueResponse>;
 
```

Comment:
@fswarbrick since account number is included in the startParams I would think just changing `acctNum` to `startParams` then see message in ZoweExplorerZosmfApi.ts

Reply:
Seems like six of one, half dozen of the other.  I wanted to minimize the number of modules I changed.  But I can do so if you'd prefer.

Reply:
@fswarbrick if those changes are made then only 1 unit test is failing and that can be fixed this way
![image](https://user-images.githubusercontent.com/49491949/122127219-ddb6d200-ce00-11eb-8158-ad03c10d0deb.png)

I will build and package your branch and test it with extenders to make sure nothing is broken in that case

Reply:
@JillieBeanSim:  Thanks for your help.  Can you point me to any documentation that explains what these tests are doing?  Apparently they count the number of parameters passed?

Also, isn't it wrong that the "getTsoParams" test is (still) returning a string, instead of IStartTsoParams, but there is no test that catches that?

Reply:
@fswarbrick we have documentation on unit tests in the wiki https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Unit-Tests, mostly testing functions and methods with mock returns for some calls.  The Object.defineProperty is mocking the call to getTsoParams returning an object with only account within in the image above, so any method in this set of unit tests that call getTsoParams will get this object back.